allure-report-face/src/helpers/splitUriList.js

```diff
@@ -0,0 +1,6 @@
+export default function splitUriList(uri) {
+    var os = require('os');
+    return uri.split(os.EOL).filter(function (el) {
+        return el.trim().length > 0 && !el.trim().startsWith('#');
+    });
+}
```

Comment:
@just-boris @baev guys, changes from last screenshot haven't been pushed yet. Merged version doesn't support comments. @just-boris if you're going to create PR with fixes, please see the following commit, which adds comments support: https://github.com/allure-framework/allure-core/commit/24de705f310e52277c0789382071eb3064f851fa Would be appreciated, if you could add corresponding updates.